lighthouse-core/test/lib/tap-target-rect-helpers-test.js

```diff
@@ -0,0 +1,246 @@
+/**
```

Comment:
can we get this filename aligned with the source one? no preference and not sure if you already had some refactoring done that makes one name easier than the other, so up to you!

Reply:
Sure, changed it to `tappable-rects-test.js`.